api/routes/index.js

```diff
@@ -12,10 +12,10 @@ export default app => {
     res.end(
       `
       API:
-      Qaris: '/api/qaris'
-      audio_files: '/api/audio_files'
-      surahs: '/api/surahs'
-      sections: '/api/sections'
+      Qaris: '/qaris'
```

Comment:
why remove `/api`?

Reply:
they weren't reflecting on the endpoints. i.e the path `/api/qari/:id` returns 404 but `/qari/:id` didn't.
Or this the case only for development environment?